R/plot.R

```diff
@@ -521,7 +521,7 @@ get_colors = function(pal, n, xc) {
 			if (all(is.na(x)) || all(range(x, na.rm = TRUE) == 0))
 				rep(1, length(x))
 			else
-				cut(x, n)
+				cut(x, breaks = quantile(x, seq(0, 1, 1/n), na.rm = TRUE))
```

Comment:
I can see that this would be useful to some, but why would it be a better default? (In the end, I guess we need a color legend, and could make this optional)